src/ansys/geometry/core/__init__.py

```diff
@@ -70,3 +70,6 @@
 
 DOCUMENTATION_BUILD: bool = os.environ.get("PYANSYS_GEOMETRY_DOC_BUILD", "false").lower() == "true"
 """Global flag for the documentation to use the proper PyVista Jupyter backend."""
+
+USE_TRACKER_TO_UPDATE_DESIGN: bool = True
```

Comment:
```suggestion
USE_TRACKER_TO_UPDATE_DESIGN: bool = False
```
Reverting to ``False`` as discussed internally - I'm glad it's working!!! Thanks @umutsoysalansys